hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTimeline.java

```diff
@@ -111,21 +115,21 @@
    * view is constructed with this timeline so that file-slice after pending compaction-requested instant-time is also
    * considered valid. A RT file-system view for reading must then merge the file-slices before and after pending
    * compaction instant so that all delta-commits are read.
-   * 
+   *
```

Comment:
Can we avoid these changes ? Might be due to your intellij setup, please check

Reply:
Sure. will fix my setup.